pyccel/commands/pyccel_config.py

```diff
@@ -0,0 +1,45 @@
+#------------------------------------------------------------------------------------------#
+# This file is part of Pyccel which is released under MIT License. See the LICENSE file or #
+# go to https://github.com/pyccel/pyccel/blob/devel/LICENSE for full license details.      #
+#------------------------------------------------------------------------------------------#
+""" Module containing scripts to manage compilation information.
+"""
+
+from .argparse_helpers import check_file_type, add_compiler_selection
+
+PYCCEL_CONFIG_DESCR = 'Compilation configuration management.'
+
+def setup_pyccel_config_parser(parser):
+    """
+    Add the pyccel config arguments to the parser.
+
+    Add the pyccel config arguments to the parser for command line arguments.
```

Comment:
For clarity I suggest quoting `pyccel config`:
```suggestion
    Add the 'pyccel config' arguments to the parser.

    Add the 'pyccel config' arguments to the parser for command line arguments.
```